flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/main/java/org/apache/flink/cdc/connectors/paimon/sink/PaimonMetadataApplier.java

```diff
@@ -73,6 +75,10 @@ public class PaimonMetadataApplier implements MetadataApplier {
 
     private Set<SchemaChangeEventType> enabledSchemaEvolutionTypes;
 
+    private static final String OPTION_DEFAULT_VALUE_PREFIX = "FIELDS";
```

Comment:
To trace the source code, We could use org.apache.paimon.CoreOptions.FIELDS_PREFIX directly.